content/en/tracing/troubleshooting/tracer_debug_logs.md

```diff
@@ -30,9 +30,41 @@ To enable debug mode for the Datadog Java Tracer, set the flag `-Ddd.trace.debug
 
 {{< programming-lang lang="python" >}}
 
-To enable debug mode for the Datadog Python Tracer, set the environment variables `DD_TRACE_DEBUG=true` and `DD_CALL_BASIC_CONFIG=true` when using `ddtrace-run`.
+To enable debug mode for the Datadog Python Tracer, review the instructions below for the specific scenario that applies to the application being traced. Logs emitted by the Python Tracer contain the name **[ddtrace]**.
+
+#### 1.3.0 or higher
+
+1. Set these environment variables to enable debug mode:
+    - `DD_TRACE_DEBUG=true`
+    - `DD_TRACE_LOGGING_RATE=0` (Optional: used for more tracer logs)
+2. To route to a log file:
+    - Configure `DD_TRACE_LOG_FILE` with the log file that tracer logs should be written to. Example: `DD_TRACE_LOG_FILE=ddtrace_logs.log`.
+    - By default, the file size is `DD_TRACE_LOG_FILE_SIZE_BYTES=15728640` or ~15 mb and one backup log file will be created. To increase the default log file size, change `DD_TRACE_LOG_FILE_SIZE_BYTES` to the desired file bytes size.
+3. To route logs to the console: 
+    - This is done by default for **Python 3** applications. 
+    - For **Python 2** applications, `logging.basicConfig()` or similar needs to be configured.
+
+#### 1.0.0 to 1.2.0
+
+1. Set these environment variables to enable debug mode:
+    - `DD_TRACE_DEBUG=true`
+    - `DD_CALL_BASIC_CONFIG=true`
+    - `DD_TRACE_LOGGING_RATE=0` (Optional: used for more tracer logs)
+2. `ddtrace` logs are routed to the console by default.
+
+
+#### 0.x
+
+1. Set these environment variables to enable debug mode::
+    - `DD_TRACE_DEBUG=true`
+    - `DD_CALL_BASIC_CONFIG=true`
```

Comment:
I think we don't actually need this envar since we use to call `basicConfig()` by default before 1.x https://github.com/DataDog/dd-trace-py/pull/3168

Reply:
this is correct, for versions <1.0 we don't need to call `DD_CALL_BASIC_CONFIG=true`